### ISSUE TITLE
Android Image.load_jpg_from_buffer()  Unable to parse normally

### ISSUE THREAD

**Issue by she-zi** [godotengine/godot#59460]
### Godot version

3.4.4 and 3.3.4

### System information

windows 11, gles 3 ,AMD Ryzen 75800H with Radeon Graphics

### Issue description

Android:
![QQ图片20220324152414](https://user-images.githubusercontent.com/57996980/159863946-eba38f41-474d-4f4b-9492-e7a0aefefd2c.jpg)

Unable to load normally

windows:
![微信图片_20220324152517](https://user-images.githubusercontent.com/57996980/159864101-03b4c771-f320-4af4-9273-a974b76f5ab5.png)

Normal loading

### Steps to reproduce

```gdscript
extends Control


# Declare member variables here. Examples:
# var a = 2
# var b = "text"

var listarr = []
# Called when the node enters the scene tree for the first time.
func _ready():
	dir_contents("/storage/emulated/0/DCIM/Screenshots/")
	dir_contents("/storage/emulated/0/Tencent/QQ_images/")
	pass # Replace with function body.
func a(filepath:String,tag):
	var f = File.new()
	f.open(filepath,File.READ)
	var buffer = f.get_buffer(f.get_len())
	f.close()
	var img = Image.new()
	if (tag == "jpg"):
		if img.load_jpg_from_buffer(buffer) !=0:
			return
	elif (tag == "png"):
		if img.load_png_from_buffer(buffer) !=0:
			return
	var texture = ImageTexture.new()
	texture.create_from_image(img)
	return texture

func dir_contents(path):
	var dir = Directory.new()
	if dir.open(path) == OK:
		dir.list_dir_begin()
		var file_name = dir.get_next()
		while file_name != "":
			yield(get_tree(),"idle_frame")
			if dir.current_is_dir():
				$Label.text +=("发现目录：" + file_name)+"\t"
#				dir_contents(path+"/"+file_name)
			else:
				
				$Label.text +=("发现文件：" + file_name)+"\t"
				if (".jpg" in file_name or ".png" in file_name ):
					var TextureRect_ =  load("res://TextureRect.tscn").instance()
					get_node("ScrollContainer/GridContainer").add_child(TextureRect_)
					TextureRect_.texture = a(path+"/"+file_name,file_name.substr(file_name.length()-3,file_name.length()-1))
			file_name = dir.get_next()
			
	else:
		$Label.text += ("尝试访问路径时出错。")+"\t"

# Called every frame. 'delta' is the elapsed time since the previous frame.
#func _process(delta):
#	pass
```

### Minimal reproduction project

[读取安卓截屏图片.zip](https://github.com/godotengine/godot/files/8339273/default.zip)

_Bugsquad Edit:_ Markup fix.

**Comment by timothyqiu** [comment]
Duplicate of #35459, see that issue for detailed information.

TLDR: `create_from_image` uses `Texture.FLAG_MIPMAP` by default, and getting rid of that flag solves the problem. (or you can try converting the image into `RGBA8`.)
